packages/conversation/src/rest/v1/contact/contact-api.ts

```diff
@@ -129,12 +130,14 @@ export class ContactApi extends ConversationDomainApi {
 
   /**
    * Get Channel Profile
-   * Get user profile from a specific channel. Only supported on &#x60;MESSENGER&#x60;, &#x60;INSTAGRAM&#x60;, &#x60;VIBER&#x60; and &#x60;LINE&#x60; channels. Note that, in order to retrieve a WhatsApp display name, you can use the Get a Contact or List Contacts operations, which will populate the &#x60;display_name&#x60; field of each returned contact with the WhatsApp display name (if the name is already stored on the server and the &#x60;display_name&#x60; field has not been overwritten by the user).
-   * @param { GetChannelProfileRequestData } data - The data to provide to the API call.
+   * Get user profile from a specific channel. Only supported on 'MESSENGER', 'INSTAGRAM', 'VIBER' and LINE' channels.
+   * Note that, in order to retrieve a WhatsApp display name, you can use the Get a Contact or List Contacts operations, which will populate the 'display_name' field of each returned contact with the WhatsApp display name (if the name is already stored on the server and the 'display_name' field has not been overwritten by the user).
+   * @param { GetChannelProfileRequestData<Recipient> } data - The data to provide to the API call.
    */
-  public async getChannelProfile(data: GetChannelProfileRequestData): Promise<GetChannelProfileResponse> {
+  public async getChannelProfile(data: GetChannelProfileRequestData<Recipient>): Promise<GetChannelProfileResponse> {
```

Comment:
No dedicated function name usage? (`getChannelProfileByContactId` and `getChannelProfileByChannelIdentity`)